src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -233,6 +233,11 @@ public virtual PinMode GetPinMode(int pinNumber)
     /// <returns>The status if the pin is open or closed.</returns>
     public bool IsPinOpen(int pinNumber)
     {
+        if (_driver == null)
```

Comment:
Why are we only making this check here as opposed to everywhere else in this class? For instance, it's not in IsPinModeSupported and that is accessing the driver

Reply:
If it is because we expect all other method's to call this, we should probably add Debug.Asserts in all of the rest of the methods that ensure that driver is in fact not null after calling IsPinOpen.

Reply:
Yep, overlooked the IsPinModeSupported method. Fixed now.